nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -152,71 +176,72 @@ Below is a table documenting [corrected filter field changes](../release-notes/v
-|                       | `site_id`           | instead of `/ipam/vlan-groups/?site_id=<uuid>`, use `site=<uuid>`                      |
-| VLAN                  | `region_id`         | instead of `/ipam/vlans/?region_id=<uuid>`, use `region=<uuid>`                        |
-|                       | `site_id`           | instead of `/ipam/vlans/?site_id=<uuid>`, use `site=<uuid>`                            |
+| Model              | Removed Filter Field | UI and API endpoints that are no longer supported in v2.X                             |
+|--------------------|----------------------|---------------------------------------------------------------------------------------|
+| CircuitTermination | `region_id`          | instead of `/circuits/circuit-terminations/?region_id=<uuid>`, use `region=<uuid>`    |
+|                    | `site_id`            | instead of `/circuits/circuit-terminations/?site_id=<uuid>`, use `site=<uuid>`        |
+| Cluster            | `region_id`          | instead of `/virtualization/clusters/?region_id=<uuid>`, use `region=<uuid>`          |
+|                    | `site_id`            | instead of `/virtualization/clusters/?site_id=<uuid>` , use `site=<uuid>`             |
+| ConfigContext      | `role_id`            | instead of `/extras/config-contexts/?role_id=<uuid>`, use `role=<uuid>`               |
+| ConsolePort        | `region_id`          | instead of `/dcim/console-ports/?region_id=<uuid>`, use `region=<uuid>`               |
+|                    | `device_id`          | instead of `/dcim/console-ports/?device_id=<uuid>`, use `device=<uuid>`               |
+| ConsoleServerPort  | `region_id`          | instead of `/dcim/console-server-ports/?region_id=<uuid>`, use `region=<uuid>`        |
+|                    | `device_id`          | instead of `/dcim/console-server-ports/?device_id=<uuid>`, use `device=<uuid>`        |
+| Device             | `region_id`          | instead of `/dcim/devices/?region_id=<uuid>`, use `region=<uuid>`                     |
+|                    | `site_id`            | instead of `/dcim/devices/?site_id=<uuid>`, use `site=<uuid>`                         |
+|                    | `manufacturer_id`    | instead of `/dcim/devices/?manufacturer_id=<uuid>`, use `manufacturer=<uuid>`         |
+|                    | `model`              | instead of `/dcim/devices/?model=<uuid>`, use `device_type=<uuid>`                    |
+|                    | `role_id`            | instead of `/dcim/devices/?role_id=<uuid>`, use `role=<uuid>`                         |
+|                    | `platform_id`        | instead of `/dcim/devices/?platform_id=<uuid>`, use `platform=<uuid>`                 |
+|                    | `secrets_group_id`   | instead of `/dcim/devices/?secrets_group_id=<uuid>`, use `secrets_group=<uuid>`       |
+|                    | `pass_through_ports` | instead of `/dcim/devices/?pass_through_ports=<bool>`, use `has_front                 |rear_ports`      |
```

Comment:
Needs fixing

Reply:
Not sure I see anything that needs to be fixed here.

Reply:
```suggestion
|                    | `pass_through_ports` | instead of `/dcim/devices/?pass_through_ports=<bool>`, use `has_front|rear_ports`      |
```